extensions/windows/cfapi/Vfs/cloudproviderregistrar.cpp

```diff
@@ -37,7 +37,8 @@ using namespace Windows::Security::Cryptography;
 #define REGKEY_AUMID L"AUMID"
 
 // Package family name, see FileExplorerExtensionPackage / Package.appxmanifest
-#define REGVALUE_AUMID L"Infomaniak.kDrive.Extension_csy8f8zhvqa20!App"
+//#define REGVALUE_AUMID L"Infomaniak.kDrive.Extension_dbrs6rk4qqhna!App" // EV
```

Comment:
It would be nice to have a comment indicating when this line should be uncommented.

Reply:
Done